platform/linux-generic/odp_queue.c

```diff
@@ -89,10 +89,10 @@ static int queue_init(queue_entry_t *queue, const char *name,
 	}
 	queue->s.type = queue->s.param.type;
 
-	queue->s.enqueue = queue_enq;
-	queue->s.dequeue = queue_deq;
-	queue->s.enqueue_multi = queue_enq_multi;
-	queue->s.dequeue_multi = queue_deq_multi;
+	queue->s.enqueue = _queue_enq;
+	queue->s.dequeue = _queue_deq;
+	queue->s.enqueue_multi = _queue_enq_multi;
+	queue->s.dequeue_multi = _queue_deq_multi;
```

Comment:
Earlier we had functions 'odp_queue_enq/odp_queue_deq/odp_queue_enq_multi/odp_queue_deq_multi', these had to be changed to 'queue_enq/queue_deq/queue_enq_multi/queue_deq_multi'. Hence these functions had to be renamed.